### PR TITLE
Tidy Fridy - Removed CategoryCard

### PR DESCRIPTION
**Pull request by davidgilbertson**
Early on there was an 'indicator card' and also a 'category card'. The latter was later removed from the design.

This removes that code.

### CODE REVIEW

File: src/components/Card/CategoryCard/CategoryCard.test.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-
-it('should match Snapshot', () => {
-  const component =  renderer.create(
-    <CategoryCard
-      color="#eee"
-    />
-  );
-
-  expect(component.toJSON()).toMatchSnapshot();
-});
```
Comment (davidgilbertson) [inline src/components/Card/CategoryCard/CategoryCard.test.js:13]: So much deleted code :(

File: src/components/CategoryOverview/CategoryOverview.scss [hunk 2]
```diff
@@ -155,7 +155,7 @@
     flex: none;
     width: 100%;
     margin-right: 0;
-    margin-top: 8px;
+    margin-top: 16px;
```
Comment (davidgilbertson) [inline src/components/CategoryOverview/CategoryOverview.scss:158]: Unrelated: gap between the 'captains pick' pair of cards on the overview page.

File: src/components/IndicatorCard/IndicatorCard.scss [hunk 3]
```diff
@@ -8,15 +8,32 @@
...
+  padding: 4px 8px 7px;
+  border-bottom-width: 6px;
+}
+
+.largeCard {
+  height: 160px;
+  padding: 4px 9px 13px;
+  border-bottom-width: 8px;
+}
+
```
Comment (davidgilbertson) [inline src/components/IndicatorCard/IndicatorCard.scss:36]: Just merging in CSS that was in `Card.scss`

File: src/constants/categories.js [hunk 4]
```diff
@@ -21,7 +21,8 @@ export const CATEGORIES = [
     shortDescription: 'Contextual indicators can help to understand why a city performs the way it does and what policies may be effective for improving economic performance and quality of life.',
     heroIndicatorId: 'population',
     overviewIndicatorIds: [
-      'population',
+      'growthRate',
+      'population10yr',
```
Comment (davidgilbertson) [inline src/constants/categories.js:25]: Unrelated, showing two population growth cards (previously it just repeated population)

File: src/constants/indicators.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ export default {
     name: 'Population',
     shortDescription: 'Total population',
     longDescription: 'This shows the population size.\n\nSource: ABS - Regional Population Growth',
-    format: '0[.]0a',
+    format: '0[.]00a',
```
Comment (davidgilbertson) [inline src/constants/indicators.js:11]: Unrelated, some populations were just "2m". This makes them "2.04" or whatever

**Review by micapam — APPROVED** [review]
AOK

**Pull request merged**